rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -536,6 +537,15 @@ upgrade_database() {
     echo " Done"
   fi
 
+
+  # - 3.1.x and 3.2.x to 4.0.0: Migration DB schema to add archive table "archivedNodeConfigurations" and "ArchivedReportsExecution" 
+  RES=$(${PSQL} -t -d rudder -c "select count(*) from information_schema.columns where lower(table_name) = 'archivednodeconfigurations'")
```

Comment:
All other command of the same kind have also a RES. So I for merging it as it, and changing all the RES to COUNT if wanted in a following release.